spec/resolver/basic_spec.rb

```diff
@@ -35,6 +35,13 @@
     should_resolve_as %w[berkshelf-2.0.7 chef-10.26 chef_app-1.0.0 json-1.7.7]
   end
 
+  it "prefers expicitly requested dependencies when resolving an index which would otherwise be ambiguous" do
```

Comment:
There is a missing `l` for `explicitly`.

Reply:
Ha, thanks! https://github.com/bundler/bundler/pull/6135